### PR TITLE
Remove components-network_time-network_time_tracker.cc.patch

### PR DESCRIPTION
**Pull request by Gyuyoung**
We don't need to keep components-network_time-network_time_tracker.cc.patch
file because it can be done by disabling kNetworkTimeServiceQuerying
in BraveMainDelegate::BasicStartupComplete().

Resolves brave/brave-browser#12085.

Resolves 

## Submitter Checklist:

- [ ] Submitted a [ticket](https://github.com/brave/brave-browser/issues) for my issue if one did not already exist.
- [ ] Used Github [auto-closing keywords](https://help.github.com/articles/closing-issues-via-commit-messages/) in the commit message.
- [ ] Added/updated tests for this change (for new code or code which already has tests).
- Verified that these changes build without errors on
  - [ ] Android
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- Verified that these changes pass automated tests (unit, browser, security tests) on
  - [ ] iOS
  - [ ] Linux
  - [ ] macOS
  - [ ] Windows
- [ ] Verified that all lint errors/warnings are resolved (`npm run lint`, `npm run gn_check`)
- [ ] Ran `git rebase master` (if needed).
- [ ] Ran `git rebase -i` to squash commits (if needed).
- [ ] Tagged reviewers and labelled the pull request as needed.
- [ ] Requested a security/privacy review as needed.
- [ ] Added appropriate QA labels (`QA/Yes` or `QA/No`) to the associated issue
- [ ] Added appropriate release note labels (`release-notes/include` or `release-notes/exclude`) to the associated issue
- [ ] Public documentation has been updated as necessary. For instance:
  - [ ] https://github.com/brave/brave-browser/wiki/Deviations-from-Chromium-(features-we-disable-or-remove)
  - [ ] https://github.com/brave/brave-browser/wiki/Proxy-redirected-URLs
  - [ ] https://github.com/brave/brave-browser/wiki/Fingerprinting-Protections
  - [ ] https://github.com/brave/brave-browser/wiki/Brave%E2%80%99s-Use-of-Referral-Codes
  - [ ] https://github.com/brave/brave-browser/wiki/Custom-Headers
  - [ ] https://github.com/brave/brave-browser/wiki/Web-compatibility-issues-with-tracking-protection
  - [ ] https://github.com/brave/brave-browser/wiki/QA-Guide

## Reviewer Checklist:

- [ ] New files have MPL-2.0 license header.
- [ ] Request a security/privacy review as needed.
- [ ] Adequate test coverage exists to prevent regressions 
- [ ] Verify test plan is specified in PR before merging to source

## After-merge Checklist:

- [ ] The associated issue milestone is set to the smallest version that the
  changes has landed on.
- [ ] All relevant documentation has been updated.

### CODE REVIEW

File: app/brave_main_delegate.cc [hunk 1]
```diff
@@ -243,6 +244,7 @@ bool BraveMainDelegate::BasicStartupComplete(int* exit_code) {
     features::kSmsReceiver.name,
     features::kVideoPlaybackQuality.name,
     features::kTabHoverCards.name,
+    network_time::kNetworkTimeServiceQuerying.name,
```
Comment (mkarolin) [inline app/brave_main_delegate.cc:247]: Should also be added to https://github.com/brave/brave-core/blob/master/app/brave_main_delegate_browsertest.cc#L63

**Review by pilgrim-brave — APPROVED** [review]

File: app/brave_main_delegate.cc (hunk 1 above)
Comment (Gyuyoung) [reply app/brave_main_delegate.cc:247]: Sorry for the late reply. I was on holidays until yesterday. ok, I add it here. Thanks.

**Review by mkarolin — APPROVED** [review]
LGTM

**Comment by Gyuyoung** [comment]
As far as I know, I can merge a PR after getting +1 among reviewers. (If not, please let me know.) And, let's revert this PR when this will cause any regression. Merging...

**Comment by kjozwiak** [comment]
Moving this into https://github.com/brave/brave-core/milestone/102 as it was merged into `master` which is currently `1.18.x` as per https://github.com/brave/brave-browser/wiki/Brave-Release-Schedule#current-channel-information.

@Gyuyoung once a PR is merged, you'll need to move it it into the appropriate milestone. In this case, moved it into https://github.com/brave/brave-core/milestone/102 as that's the current `Nightly`/`master` channel.

**Comment by Gyuyoung** [comment]
> @Gyuyoung once a PR is merged, you'll need to move it it into the appropriate milestone. In this case, moved it into https://github.com/brave/brave-core/milestone/102 as that's the current Nightly/master channel.

Ah, sorry for missing setting the milestone. Let me set the milestone next time.

**Pull request merged**